ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form.view.js

```diff
@@ -104,6 +104,32 @@ module.exports = Marionette.LayoutView.extend({
         user.getQuerySettings().set('type', 'text')
         break
       case 'custom':
+        let sorts =
+          this.model.get('querySettings') &&
+          this.model.get('querySettings').sorts
+        if (sorts) {
+          sorts = sorts.map(sort => ({
+            attribute: sort.split(',')[0],
```

Comment:
I know this isn't due to your changes, but it's a little funky that we're parsing individual sorts as strings.  I'm guessing this is how the backend is storing them at the moment.